### ISSUE TITLE
hange backup folder structure

### ISSUE THREAD

**Issue by akalongman** [akalongman/sublimetext-autobackups#28]
YEAR/MONTH/DAY/PATH

**Comment by adongy** [comment]
Can i take this issue ?

**Comment by blake-mealey** [comment]
I might be able to help with this. I was looking at the code and seemed to me I would just have to update [this line](https://github.com/akalongman/sublimetext-autobackups/blob/master/autobackups/paths_helper.py#L59) so that instead of just the date, it adds `'/'+year+'/'+month+'/'+day`. Does that seem about right to you?

**Comment by akalongman** [comment]
You must also test carefully for make sure everything is works

**Comment by blake-mealey** [comment]
Of course! Well I'll give this a shot tomorrow but I might get busy this week.
